### DISCUSSION TITLE
How could I scale the image up?

### DISCUSSION THREAD

**Discussion by xnuohz** [Kanaries/pygwalker#293] [Q&A] [answered]
It's toooooo small.
![image](https://github.com/Kanaries/pygwalker/assets/17878022/35b30f6b-f026-456e-a25e-fe5f88c013d2)

**Comment by longxiaofei** [comment, marked as answer]
![image](https://github.com/Kanaries/pygwalker/assets/28337703/cfd32b90-2a67-4375-90a9-ec5b470e72aa)

**Reply by xnuohz** [reply]
Thank you!
